router/batchrouter/handle_async.go

```diff
@@ -696,6 +697,12 @@ func (brt *Handle) setMultipleJobStatus(asyncOutput common.AsyncUploadOutput, at
 		panic(err)
 	}
 	brt.updateProcessedEventsMetrics(statusList)
+	rmetrics.DecreasePendingEvents(
+		"batch_rt",
+		workspaceID,
+		brt.destType,
+		float64(len(asyncOutput.AbortJobIDs)+len(asyncOutput.SucceededJobIDs)),
```

Comment:
Why aren't we using `len(completedJobsList)` instead? With this we might be missing scenarios, such as failed jobs whose retry limit has been reached, etc.

Reply:
missed it!
ty